Tests/L0/Gradle/_suite.ts

```diff
@@ -23,20 +38,20 @@ describe('gradle Suite', function() {
 	
     //TODO: The 'Test Run Title' and 'Code Coverage Tool' fields are 
     //      not used by the NodeJS task currently and so are not tested.
-	
+
 
     it('run gradle with all default inputs', (done) => {
         setResponseFile('gradleGood.json');
-
-        var tr = new trm.TaskRunner('gradle');
+    
+        var tr = new TaskRunner('gradle');
```

Comment:
Importing trm.TaskRunner directly using Typescript. 

Also applying Boy Scout principle - I may as well improve their code while I'm neatening up mine, though this is inflating the number of lines changed.